static/styles/a11y-dark.css

```diff
@@ -0,0 +1,98 @@
+/*!
```

Comment:
issue: please add a license header

(saying it is under the MIT license, linking to the source, and linking to the full text of the MIT license, should be fine)

Reply:
I guess the license is BSD-3-Clause?

https://github.com/highlightjs/highlight.js/blob/main/LICENSE

Reply:
Unless other licenses apply to the themes and I missed it.